### ISSUE TITLE
Provider 1.0.1 - Required to specify leasedays even if there's a default specified in the blueprint

### ISSUE THREAD

**Issue by hobovirtual** [vmware-archive/terraform-provider-vra7#64]
**vRA 7.x version**
vRealize Automation 7.6

**Terraform version**
v0.12.24

**terraform-provider-vra7 plugin version**
The terraform-provider-vra7 plugin version 1.0.1

**Describe the bug**
Issue #60 fixed the cluster requirement but not the leasedays

**To Reproduce**
1. maint.tf content

```
provider "vra7" {
    username = var.vra_username
    password = var.vra_password
    tenant = var.vra_tenant
    host = var.vra_host
    insecure = true
}

resource "vra7_deployment" "deployment" {
  catalog_item_name = "dev vm"              
  reasons = "provide a reason"
  description = "provide a description"
  //lease_days = 2

  resource_configuration {
    component_name = "vm1"

    //Please Provide the following information for the bootstrap node component
    configuration = {
      "custom.midname" = "BASE"
    }
  } 
}
```

2. Terraform command
terraform apply

3. Error
Error: The catalog item request failed with error vRealize API: [{Code:20117 Mes
sage:The data specified within the request is invalid. SystemMessage:The data sp
ecified within the request is invalid.} {Code:11002 Message:The value of the '_l
easeDays' field cannot be less than 2. SystemMessage:The value of the field with
 id _leaseDays cannot be less than 2.} {Code:400 Message: SystemMessage:}]

**Expected behavior**
Don't declare property value when default exist in the blueprint

**Desktop (please complete the following information):**
 - OS: Windows 2016 and CentOS 7

**Comment by Prativa20** [comment]
@hobovirtual Will look into it, thanks!

**Comment by Prativa20** [comment]
Fixed #78
